src/main/java/survey/Survey.java

```diff
@@ -1,17 +1,34 @@
 package survey;
 
-import java.util.ArrayList;
+import javax.persistence.*;
+import java.util.Collection;
 
+@Entity
 public class Survey {
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Id
     private Integer id;
-    private ArrayList<Question> questions = new ArrayList<Question>();
 
-    public Survey(){}
+    private String name;
 
-    public Survey(ArrayList<Question> questions){
+    @OneToMany(cascade = CascadeType.ALL)
+    private Collection<Question> questions;
+
+    public Survey(String name){ this.name = name; }
+
+    public Survey(String name, Collection<Question> questions){
```

Comment:
Most of the methods are missing Javadocs/comments

Reply:
The file size triples because of the comments and getters and setters. It is so pointless.